### ISSUE TITLE
Cannot reproduce the pretraining experiment on K400 dataset

### ISSUE THREAD

**Issue by Raion-Shin** [SwinTransformer/Video-Swin-Transformer#98]
Hi, I tried to reproduce the swin-tiny pretraining with [the config you provided](https://github.com/SwinTransformer/Video-Swin-Transformer/blob/master/configs/recognition/swin/swin_tiny_patch244_window877_kinetics400_1k.py), but the validation result doesn't match your paper's result at all. The result I get is 55.28% top1-acc and 79.28% top5-acc. Is there anything missed?

**Comment by innat** [comment]
I translated to keras 3 and using torch backend, the score seems well. 

```
 Acc1=77.690, Acc5=93.297
```

[Notebook](https://github.com/innat/VideoSwin/blob/main/guides/eval_benchmark/kerascv-kinetics-400-evaluation-in-pytorch.ipynb).

You probably should check your dataloader and frame size and normalizaiton stuff.
